tez-common/src/main/java/org/apache/tez/common/AsyncDispatcher.java

```diff
@@ -180,6 +196,9 @@ protected void dispatch(Event event) {
       } else {
         throw new Exception("No handler for registered for " + type);
       }
+    } catch (InterruptedException ie) {
```

Comment:
Can you add this within Throwable block (with instance of checks)? Rest lgtm.

Reply:
thanks, it's done